### PR TITLE
Week 02, 03 and 04

### PR DESCRIPTION
**Pull request by chrsmendes**

### CODE REVIEW

**Review by chrsmendes — COMMENTED** [review]
Your index.html page is not in the root directory. Can you please fix that so I can analyze your week 02 performance?


image

File: wwr/about.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    <link rel="stylesheet" href="styles/rafting.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&amp;display=swap"
+      rel="stylesheet"
+    />
+    <title>White Water Rafting</title>
```
Comment (chrsmendes) [inline wwr/about.html]: Please fix the title according to the course audio tool:

image

You should add "About Us" along with the name of the rafting company in the title, like this:

```html
<title>About Us | Rapids</title>
```

File: wwr/about.html
Comment (chrsmendes) [inline wwr/about.html:1]: You are missing meta author and description. Please add in the head of your html:

image

```html
<meta name="author" content="[YOUR NAME]">
<meta name="description" content="[A DESCRIPTION OF THE WEBSITE]">
```

Comment (chrsmendes) [inline wwr/about.html:1]: You are missing the `main` tag inside the body tag for the main content. Additionally, you are not using the two required `section` tags, and each of these sections should include an `h2` heading.

image

Please make sure to carefully follow the instructions for the activity.

**Add the `main` tag in the body as specified in these instructions:**
[Add Main Tag Instruction](https://byui-cse.github.io/wdd130-ww-course/week03/prove-rafting-aboutus.html#:~:text=In%20the%20body%2C%20add%20these%20three%20main%20elements%3A%20header%2C%20main%2C%20and%20footer)

**For the section tags and `h2` heading instructions, refer to:**
[Section Tags Instruction](https://byui-cse.github.io/wdd130-ww-course/week03/prove-rafting-aboutus.html#:~:text=A%20section%20element,layout%20the%20images.)

Comment (chrsmendes) [inline wwr/about.html:1]: You are missing the page structure, which should include the `header`, `main`, and `footer` elements inside the body.

image

Please remember to read and follow all instructions carefully. Do not assume anything that is not explicitly required for you to complete the task. Keep in mind that reading and understanding instructions is a crucial skill for any developer.

**Add the `header`, `main`, and `footer` tags in the body as specified in these instructions:**
[Add Main Tag Instruction](https://byui-cse.github.io/wdd130-ww-course/week03/prove-rafting-aboutus.html#:~:text=In%20the%20body%2C%20add%20these%20three%20main%20elements%3A%20header%2C%20main%2C%20and%20footer)

File: wwr/about.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    />
+    <link rel="stylesheet" href="styles/rafting.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&amp;display=swap"
+      rel="stylesheet"
+    />
```
Comment (chrsmendes) [inline wwr/about.html:24]: Why are you importing the Roboto font family? I believe you are only using Merriweather. Please remove all Google Fonts links and use only the following:

```html
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300..800;1,300..800&display=swap" rel="stylesheet">
```

**Review by chrsmendes — COMMENTED** [review]
Please check the comments and fix them in order to finish your week 2, 3 and 4 assignments!

File: wwr/about.html (hunk 1 above)
Comment (tiagogago) [reply wwr/about.html]: feito

File: wwr/about.html
Comment (tiagogago) [reply wwr/about.html:1]: feito

Comment (tiagogago) [reply wwr/about.html:1]: feito

Comment (tiagogago) [reply wwr/about.html:1]: feito

Comment (tiagogago) [reply wwr/about.html:1]: feito

Comment (tiagogago) [reply wwr/about.html:1]: feito

Comment (tiagogago) [reply wwr/about.html:1]: feito

File: wwr/about.html (hunk 2 above)
Comment (tiagogago) [reply wwr/about.html:24]: feito

**Pull request merged**